users/ignoring.rst

```diff
@@ -98,26 +124,15 @@ The ``.stignore`` file contains a list of file or path patterns. The
    they are preventing directory deletion. This prefix should be used by any OS
    generated files which you are happy to be removed.
 
+   .. note::
```

Comment:
I'd leave this one outside of the bullet list, as it applies to the last two points equally.